plugin/hpsa/hpsa.py

```diff
@@ -909,3 +933,223 @@ def volume_ident_led_clear(self, volume, flags=Client.FLAG_RSVD):
                 "Volume not found")
 
         return None
+
+    @_handle_errors
+    def batteries(self, search_key=None, search_value=None,
+                  flags=Client.FLAG_RSVD):
+        lsm_batteries = []
+        ctrl_all_show = self._sacli_exec(
+            ["ctrl", "all", "show", "config", "detail"])
+
+        for ctrl_name in ctrl_all_show.keys():
+            ctrl_data = ctrl_all_show[ctrl_name]
+            if 'Battery/Capacitor Count' not in ctrl_data or \
```

Comment:
The string `'Battery/Capacitor Count'` gets used a number of times, I would suggest placing it in a constant so when/if it needs to get changed it will be easier.